src/scripts/temporaryBattle/TemporaryBattleList.ts

```diff
@@ -282,6 +282,126 @@ TemporaryBattleList['Blue 6'] = new TemporaryBattle(
         imageName: 'Blue2',
     }
 );
+// Kanto Christmas Temporary Battles, based on Blue 2
+TemporaryBattleList['Santa Jynx 1'] = new TemporaryBattle(
+    'Santa Jynx 1',
+    [
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+    ],
+    'Huh ? They are gone by the routes now.',
+    [
+        new SpecialEventRequirement('Merry Christmas!'),
+        new ItemRequirement(11, 'Christmas_present', GameConstants.AchievementOption.less),
+        new TemporaryBattleRequirement('Santa Jynx 2', 1, GameConstants.AchievementOption.less),
+        new TemporaryBattleRequirement('Santa Jynx 3', 1, GameConstants.AchievementOption.less),
+        new TemporaryBattleRequirement('Santa Jynx 4', 1, GameConstants.AchievementOption.less),
+    ],
+    undefined,
+    {
+        displayName: 'Santa Jynx Band',
+        firstTimeRewardFunction: () => {
+            (ItemList.Christmas_present as ChristmasPresent).gain();
+        },
+        resetDaily: true,
+        hideTrainer: true,
+        imageName: 'Bill\'s Grandpa without Eevee',
+    }
+);
+TemporaryBattleList['Santa Jynx 2'] = new TemporaryBattle(
+    'Santa Jynx 2',
+    [
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+        new GymPokemon('Santa Jynx', 12240, 20),
+    ],
+    'Huh ? They are gone by the routes now.',
+    [
+        new SpecialEventRequirement('Merry Christmas!'),
+        new ItemRequirement(27, 'Christmas_present', GameConstants.AchievementOption.less),
+        new ItemRequirement(11, 'Christmas_present'),
+        new TemporaryBattleRequirement('Santa Jynx 1', 1, GameConstants.AchievementOption.less),
```

Comment:
The Santa Jynx 2, 3, and 4 temp battles don't appear because these previous temp battle requirements are set to `less` when they should be `more` (or left as default).

Reply:
Those are daily tempbattles.
The tempbattle 2 unlocks on day 2 if you collected enough presents, tempbattle 3 on day 3 with more presents and tempbattle 4 on every next day after more presents have dropped. I therefore meant `less` to prevent a battle to unlock on the same day as the previous one.

Reply:
Ahhhh, okay. That makes sense. I thought they were supposed to become available one after another